extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -476,6 +476,9 @@
     "ampPagePing": "https://!collectorHost/i?p=web&tv=amp-1.0.2&e=ue&ue_pr=%7B%22schema%22%3A%22iglu%3Acom.snowplowanalytics.snowplow%2Funstruct_event%2Fjsonschema%2F1-0-0%22%2C%22data%22%3A%7B%22schema%22%3A%22iglu%3Adev.amp.snowplow%2Famp_page_ping%2Fjsonschema%2F1-0-0%22%2C%22data%22%3A%7B%22scrollLeft%22%3A_scroll_left_%2C%22scrollWidth%22%3A_scroll_width_%2C%22viewportWidth%22%3A_viewport_width_%2C%22scrollTop%22%3A_scroll_top_%2C%22scrollHeight%22%3A_scroll_height_%2C%22viewportHeight%22%3A_viewport_height_%2C%22totalEngagedTime%22%3A_total_engaged_time_%7D%7D%7D&url=_ampdoc_url_&page=_title_&res=_screen_width_x_screen_height_&dtm=_timestamp_&tz=_timezone_code_&aid=!appId&cd=_screen_color_depth_&cs=_document_charset_&lang=_browser_language_&refr=_document_referrer_&vp=_viewport_width_x_viewport_height_&ua=_user_agent_&ds=_scroll_width_x_scroll_height_&uid=!userId&co=%7B%22schema%22%3A%22iglu%3Acom.snowplowanalytics.snowplow%2Fcontexts%2Fjsonschema%2F1-0-0%22%2C%22data%22%3A%5B%7B%22schema%22%3A%22iglu%3Adev.amp.snowplow%2Famp_id%2Fjsonschema%2F1-0-0%22%2C%22data%22%3A%7B%22ampClientId%22%3A%22_client_id(_sp_ampid)_%22%2C%20%22domainUserid%22%3A%20%22_if(_substr(_query_param(_sp)_%2C0%2C36)_%2C_substr(_query_param(_sp)_%2C0%2C36)_%2C_if(_linker_param(sp_amp_linker%2C_sp_duid)_%2C_linker_param(sp_amp_linker%2C_sp_duid)_%2C_cookie(_sp_duid)_)_)_%22%2C%20%22userId%22%3A%20%22!userId%22%7D%7D%2C_replace(!customContexts%2C%5E%2C*(.%2B%3F)%2C*%24%2C%241%2C)_%7B%22schema%22%3A%22iglu%3Adev.amp.snowplow%2Famp_web_page%2Fjsonschema%2F1-0-0%22%2C%22data%22%3A%7B%22ampPageViewId%22%3A%22_page_view_id_64_%22%7D%7D%5D%7D"
   },
+  "taboola": {
+    "url": "https://3p.ampproject.net/custom/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}"
```

Comment:
@micajuine-ho I need this file https://3p.ampproject.net/custom/amp-analytics-taboola.html to be checked in or hosted at 3p.ampproject.net. Is there any documentation on how to do it ?

Reply:
hi @micajuine-ho, what would you recommend us to do here ?

Reply:
Why does you need it to be hosted on that URL rather than `https://cdn.ampproject.org/v0/analytics-vendors/taboola.json`?

Reply:
@micajuine-ho part of the config in taboola.json is to load that html file which in turn loads our tracking javascript

Reply:
Are you talking about this?   
```
"requests": {
    "url": "https://3p.ampproject.net/custom/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}"
  },
```

Reply:
@micajuine-ho yeah

Reply:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#send-data-in-house-

Urls in `request` do not have to be `3p.ampproject...`

Instead you can have the url point to your own servers and handle the collection of analytics there.

Reply:
@micajuine-ho got it. let me do that. Thanks

Reply:
<img width="1270" alt="CORS-ISSUE" src="https://user-images.githubusercontent.com/78616025/111179467-593a8600-8569-11eb-9c34-321e1132d80e.png">
@micajuine-ho I updated the config to point to it hosted from our servers. This is generating CORB issue as shown in the attached image.

Reply:
Are you overwriting the transport API in the config?

Reply:
![taboola-json-changes](https://user-images.githubusercontent.com/78616025/111187898-b6d2d080-8571-11eb-8224-761fc0881f6d.png)
@micajuine-ho here are the changes that I have. Is this what you are referring to ?

Reply:
Where are you seeing this error? I'm testing manually and I see the request being sent.

Reply:
@micajuine-ho Running it locally http://localhost:8000/examples/analytics-vendors.amp.html?type=taboola

Reply:
As you can see, on that test page, the CORB error is occurring with all the vendors. If you isolate your analytics example by moving it to it's own test page, you won't see the error anymore.

Reply:
![image](https://user-images.githubusercontent.com/30711678/111213283-d677f200-858d-11eb-91b8-da386815d1a9.png)

Reply:
@micajuine-ho yeah. I tested it on one of our publisher page and works fine. Thanks

Reply:
@micajuine-ho what are the next steps for getting it merged ?